### PR TITLE
DOCSP-41412 fixing ndjson broken link

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Fixing ndjson broken link 

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/mongodb-shell/DOCSP-41412/logs/

## JIRA

https://jira.mongodb.org/browse/DOCSP-41412

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66902a32fb8bd13b841eb27a

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by sarah-olson-mongodb — APPROVED** [review]
LGTM @ltran-mdb2 and thanks for the super speedy fix!

**Comment by kirkdotcam** [comment]
TY @ltran-mdb2!

**Review by gribnoysup — APPROVED** [review]

File: source/logs.txt [hunk 1]
```diff
@@ -12,10 +12,10 @@ Retrieve Shell Logs
    :depth: 1
    :class: singlecol
 
-|mdb-shell| uses `ndjson <http://ndjson.org/>`__ to store session logs.
-Starting in :binary:`mongosh` version 1.0.5, the log |mdb-shell|
-format is updated to match the MongoDB server log format. For details,
-see :ref:`log-messages-ref`. 
+|mdb-shell| uses `ndjson <https://github.com/ndjson/ndjson-spec>`__ to
```
Comment (addaleax) [inline source/logs.txt:15]:
```suggestion
|mdb-shell| uses `newline-delimited JSON <https://github.com/ndjson/ndjson-spec>`__ to
```

Just a suggestionm, but if we spell out what `ndjson` stands for, this format almost becomes self-explanatory anyway 🙂

**Review by addaleax — APPROVED** [review]

File: source/logs.txt (hunk 1 above)
Comment (ltran-mdb2) [reply source/logs.txt]: Made this change! Thank you @addaleax !

**Pull request merged**